echopype/tests/calibrate/test_ecs.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+from datetime import datetime
+
+from echopype.calibrate.ecs_parser import ECSParser
+
+data_dir = Path("./echopype/test_data/ecs")
+
+
+def test_convert_ecs():
+    # Test converting an EV calibration file (ECS)
+    ecs_path = data_dir / "Summer2017_JuneCal_3freq_mod.ecs"
+
+    ecs = ECSParser(ecs_path)
+    ecs.parse()
+
+    # Spot test parsed outcome
+    assert ecs.data_type == "SimradEK60Raw"
+    assert ecs.version == "1.00"
+    assert ecs.file_creation_time == datetime(
+        year=2015, month=6, day=19, hour=23, minute=26, second=4
+    )
+    assert ecs.parsed_params["fileset"]["SoundSpeed"] == 1496
+    assert ecs.parsed_params["fileset"]["TvgRangeCorrection"] == "BySamples"
+    assert ecs.parsed_params["fileset"]["TvgRangeCorrectionOffset"] == 2
+    assert ecs.parsed_params["sourcecal"]["T1"]["MinorAxisAngleOffset"] == -0.18
+    assert ecs.parsed_params["sourcecal"]["T2"]["MajorAxis3dbBeamAngle"] == 6.85
+    assert ecs.parsed_params["sourcecal"]["T3"]["Ek60TransducerGain"] == 26.55
+    assert ecs.parsed_params["localcal"]["MyCal"]["TwoWayBeamAngle"] == -17.37
```

Comment:
@leewujung Sorry I didn't see my tagged name until now. Do you have an expected params? Theses asserts assumes that you have all of these parameters in there, and you said, this doesn't test for every single params. So, I suggest using a dictionary comparison so that way, you can really see if the parsed_params are showing an expected outcome. Thanks.

Reply:
Thanks @lsetiawan, I agree it'll be better to test for everything. The ones I selected here are those that are more prone to error, like negative numbers etc.

Reply:
@imranmaj : would you be able to help implement the exhaustive test @lsetiawan suggested? This is straightforward, but I am a bit out of bandwidth this week. Thanks!